src/deepsparse/image_classification/README.md

```diff
@@ -0,0 +1,244 @@
+and accuracy.
+
+To learn about sparsification in more detail, refer to [SparseML docs](https://docs.neuralmagic.com/sparseml/)
+
+### HTTP Server
+
+As an alternative to Python API, the DeepSparse inference server allows you to 
+serve ONNX models and pipelines in HTTP. Both configuring and making requests 
+to the server follow the same parameters and schemas as the Pipelines enabling 
+simple deployment. Once launched, a `/docs` endpoint is created with full 
+endpoint descriptions and support for making sample requests.
+
+Example deployment using a 95% pruned resnet50 is given below
+For full documentation on deploying sparse image classification models with the
+DeepSparse Server, see the [documentation](https://github.com/neuralmagic/deepsparse/tree/main/src/deepsparse/server).
+
+##### Installation
+
+The deepsparse server requirements can be installed by specifying the `server` 
+extra dependency when installing DeepSparse.
+
+```bash
+pip install deepsparse[server]
+```
+
+## Deployment Use Cases
+
+The following section includes example usage of the Pipeline and server APIs for
+various image classification models. 
+
+[List of Image Classification SparseZoo Models](https://sparsezoo.neuralmagic.com/?domain=cv&sub_domain=classification&page=1)
+
+
+#### Python Pipeline
+
+```python
+from deepsparse import Pipeline
+cv_pipeline = Pipeline.create(
+  task='image_classification', 
+  model_path='zoo:cv/classification/resnet_v1-50/pytorch/sparseml/imagenet/pruned95-none',  # Path to checkpoint or SparseZoo stub
+)
+input_image = "my_image.png" # path to input image
+inference = cv_pipeline(images=input_image)
+```
+
+#### HTTP Server
+
+Spinning up:
+```bash
+deepsparse.server \
+    --task image_classification \
+    --model_path "zoo:cv/classification/resnet_v1-50/pytorch/sparseml/imagenet/pruned95-none" \
+    --port 5543
+```
+
+Making a request:
+`imagenet_client.py`
+```python
+from glob import glob
+import pathlib
+import click
+import requests
+import cv2
+default_url = "http://localhost:5543/predict"
+@click.command()
+@click.option(
+    '--url',
+    default=default_url,
+    help='The URL to the server',
+    show_default=True,
+)
+@click.option(
+    '--data',
+    '--data',
+    default="data",
+    type=click.Path(exists=True, file_okay=True, dir_okay=True),
+)
+@click.option(
+    '--max_samples',
+    '--max-samples',
+    default=100,
+    type=int,
+    help='The maximum number of samples to test',
+)
+def main(url, data, max_samples):
+    input_path = pathlib.Path(data)
+    input_file_names = (
+        glob(f"{data}/*.jpg") + glob(f"{data}/*.jpeg") + glob(f"{data}/*.JPEG")
+        if input_path.is_dir()
+        else [data]
+    )
+    for index, image_path in enumerate(input_file_names):
+        if index >= max_samples:
+            break
+        image = cv2.imread(image_path)
+        image = cv2.resize(image, (224, 224))
+        payload = {
+            "images": [image.tolist()]
```

Comment:
should upload as file, not list for main example since that is the preferred pathway